internal/dbsqlc/job_list.go

```diff
@@ -0,0 +1,118 @@
+package dbsqlc
```

Comment:
So I think my main (big) nit is mostly that this should live somewhere besides `dbsqlc` — I think that it's a little too confusing that this code _looks_ like it's generated, but it's not generated, and that `dbsqlc` shares a mix of generated and non-generated code. IMO, we should put it somewhere else even if it looks almost exactly like `dbsqlc` code.

Reply:
I moved this into `db`, not sure if you have anything better in mind. It's getting a little confusing with `dbsqlc` + `dbadapter` + `db` all in there 🤔